Client/storyteller_test/storyteller_test.cpp

```diff
@@ -39,8 +39,10 @@ int main(int argc, char**argv)
         sleep(10);
         uint64_t log_time = get_chrono_timestamp();
         chl::LogEvent event(1, log_time, 7, i, "line_" + std::to_string(i));
-        std::cout << "generated_event {" << event.storyId << ":" << event.time() << ":" << event.clientId << ":"
-                  << event.index() << "}" << std::endl;
+        LOG_DEBUG("generated_event {}:{}:{}:{}", event.storyId, event.time(), event.clientId, event.index());
```

Comment:
In my case, log.h needs to be included to build this file.

Reply:
Done